cmd/kms-server/startcmd/params.go

```diff
@@ -112,6 +112,11 @@ const (
 	disableAuthFlagUsage = "Disables authorization. Possible values: [true] [false]. Defaults to false. " +
 		commonEnvVarUsageText + disableAuthEnvKey
 
+	disableHTTPSIGEnvKey    = "KMS_GNAP_HTTPSIG_DISABLE"
+	disableHTTPSIGFlagName  = "disable-HTTPSIG"
+	disableHTTPSIGFlagUsage = "Disables GNAP HTTPSIG validation. Possible values: [true] [false]. Defaults to true. " +
```

Comment:
its say default to true

Reply:
it should be false, i will update, thanks

Reply:
updated